source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -1,13 +1,15 @@
 #include <gtest/gtest.h>
 
 #include "device_server.h"
+#include "nidcpower/nidcpower_client.h"
```

Comment:
@doshirohan I added a client generator in #326, that's what the new code in this class is using.